File: NavitiaSDKSandbox/NavitiaSDKSandbox/NavitiaFeatures.swift
//
// Created by Chakkra CHAK on 25/04/2017.
// Copyright (c) 2017 Chakkra CHAK. All rights reserved.
//

import Foundation

open class NavitiaFeatures {
    let navitiaConfiguration:NavitiaConfiguration
    // <Generated>
    open let autoComplete: EndpointPlaces
    // </ Generated>

    public init(navitiaConfiguration:NavitiaConfiguration) {
        self.navitiaConfiguration = navitiaConfiguration
        // <Generated>
        self.autoComplete = EndpointPlaces(navitiaConfiguration: self.navitiaConfiguration)
        // </ Generated>
    }
}

File: android/NavitiaSDKSandbox/src/main/java/org/kisio/labs/NavitiaSDKSandbox/NavitiaFeatures.java
package org.kisio.labs.NavitiaSDKSandbox;

import org.kisio.labs.NavitiaSDKSandbox.Places.EndpointPlaces;

/**
 * Created by cchak on 27/04/2017.
 */
public class NavitiaFeatures {
    private NavitiaConfiguration navitiaConfiguration;
    private EndpointPlaces autoComplete;

    public NavitiaConfiguration getNavitiaConfiguration() {
        return navitiaConfiguration;
    }

    public EndpointPlaces getAutoComplete() {
        return autoComplete;
    }

    public NavitiaFeatures(NavitiaConfiguration navitiaConfiguration) {
        this.navitiaConfiguration = navitiaConfiguration;
        this.autoComplete = new EndpointPlaces();
    }
}
